packages/svelte/src/internal/client/dom/blocks/snippet.js

```diff
@@ -38,16 +38,18 @@ export function snippet(node, get_snippet, ...args) {
 			snippet_effect = null;
 		}
 
-		if (snippet) {
-			snippet_effect = branch(() => /** @type {SnippetFn} */ (snippet)(anchor, ...args));
-		}
+		snippet_effect = branch(() => /** @type {SnippetFn} */ (snippet)(anchor, ...args));
```

Comment:
Will we throw a more descriptive error at dev time here? Like, before the `snippet_effect = ...` line a `if (DEV && !snippet) e.blabla()`